docs_zh-CN/tools/onnx2tensorrt.md

```diff
@@ -0,0 +1,76 @@
+# ONNX 转 TensorRT（试验性的）
+
+<!-- TOC -->
+
+- [ONNX 转 TensorRT（试验性的）](#onnx-tensorrt)
+  - [如何将模型从 ONNX 转换到 TensorRT](#id1)
+    - [准备工作](#id2)
+    - [使用方法](#id3)
+  - [支持转换至 TensorRT 的模型列表](#tensorrt)
+  - [提示](#id4)
+  - [常见问题](#id5)
+
+<!-- TOC -->
+
+## 如何将模型从 ONNX 转换到 TensorRT
+
+### 准备工作
+
+1. 请参照 [install.md](https://mmclassification.readthedocs.io/en/latest/install.html#install-mmclassification) 从源码安装 MMClassification。
+2. 使用我们的工具 [pytorch2onnx.md](./pytorch2onnx.md) 将 PyTorch 模型转换至 ONNX。
+
+### 使用方法
+
+```bash
+python tools/deployment/onnx2tensorrt.py \
+    ${MODEL} \
+    --trt-file ${TRT_FILE} \
+    --shape ${IMAGE_SHAPE} \
+    --workspace-size {WORKSPACE_SIZE} \
+    --show \
+    --verify \
+```
+
+所有参数的说明：
+
+- `model` : ONNX 模型的路径。
+- `--trt-file`: TensorRT 引擎文件的输出路径。如果没有指定，默认为 `tmp.trt`。
+- `--shape`: 模型输入的高度和宽度。如果没有指定，默认为 `224 224`。
+- `--workspace-size` : 构建 TensorRT 引擎所需要的 GPU 空间大小，单位为 GiB。如果没有指定，默认为 `1` GiB。
+- `--show`: 是否展示模型的输出。如果没有指定，默认为 `False`。
+- `--verify`: 是否使用 ONNXRuntime 和 TensorRT 验证模型转换的正确性。如果没有指定，默认为`False`。
+
+示例：
+
+```bash
+python tools/onnx2tensorrt.py \
```

Comment:
The file path has been moved to `mmclassification/tools/deployment/`, we may check this and the English version.